.github/workflows/notes.md

```diff
@@ -6,8 +6,26 @@ ${NVIM_VERSION}
 
 ### Windows
 
-1. Extract **nvim-win64.zip**
-2. Run `nvim-qt.exe`
+#### Zip
+
+1. Download **nvim-win64.zip**
+2. Extract the zip.
+3. Run `nvim-qt.exe`
+
+#### MSI
+
+1. Download **nvim-win64.msi**
+2. Run the MSI
+3. Add the Neovim location to your path.
+   - Default location is `C:\Program Files\Neovim`
+4. Search and run `nvim-qt.exe` or run `nvim.exe` on your CLI of choice.
+
+#### NSIS
```

Comment:
what's the use-case for having both NSIS and MSI? if there is not a strong case we should choose one. Giving users unnecessary choices is bad UI.

Reply:
@justinmk thanks for the review and kind words. This PR's been merged already 😉. Being that we haven't seen a release yet since this change and the general consensus is that multiple installations are bad, do you want to make a captain's call or discuss whether we should have only one installer? If we're going to rip one out, I'd rather not have to incur the cost to downstream consumers by making them switch to the other in the 0.8 release. I'm not sure when the freeze is on 0.7. Is there still time? 

@mjlbach do you have any thoughts on this?

Reply:
I don't see a major downstream risk if we change the installers, but we should just pick one.

- My impression is that MSI is preferred / more flexible / has more support with default Windows CLI tooling.
    - I see that [wix file](https://github.com/neovim/neovim/issues/12571#issuecomment-1093673642) might be required, but we could skip that for now and accept the $PATH limitation (with a tracking issue so that someone can pick up that work).

I also saw somewhere that [MSIX is preferred](https://github.com/neovim/neovim/issues/12571#issuecomment-1093673642) but that can also wait until someone picks it up.

Reply:
My only thoughts are do exactly what Justin says :) We can merge a PR that changes the installers prior to 0.7 being released.

Reply:
Sure thing @mjlbach. @justinmk the NSIS installer is a nice out-of-the-box experience because of the path addition but if you think that the cost of implementing the patch file is worth it over the long term, I can rip the EXE out and leave the MSI. Unfortunately, CPack doesn't have MSIX support yet. I have no experience with MSIX, so can't really comment on it 😅.

Reply:
> NSIS installer is a nice out-of-the-box experience because of the path addition but if you think that the cost of implementing the patch file is worth it over the long term,

I'm imagining the cost of the WIX patch to be 1 file and maybe a couple lines of configuration. If that's accurate then I'd say let's drive towards that. $PATH will have to wait until it bothers someone enough.

> Unfortunately, CPack doesn't have MSIX support yet

waiting for cpack is totally reasonable (and preferable to a bunch of scripting)

Reply:
Alrighty, I'll get to it and tag you when it's ready for review @justinmk 😁

Reply:
Addressed discussion in #18069.